src/test/java/seedu/address/logic/commands/FilterTagCommandTest.java

```diff
@@ -33,14 +33,14 @@ public void equals() throws Exception {
         PersonHasTagPredicate secondPredicate =
                 new PersonHasTagPredicate(TestUtil.stringsToTags(Collections.singletonList("second")));
 
-        FilterCommand sortFirstCommand = new FilterCommand(firstPredicate);
-        FilterCommand sortSecondCommand = new FilterCommand(secondPredicate);
+        FilterTagCommand sortFirstCommand = new FilterTagCommand(firstPredicate);
```

Comment:
Oops missed out in your first pr but variable names and type names are inconsistent.